tests/unit_tests/test_github_service.py

```diff
@@ -0,0 +1,92 @@
+from unittest.mock import patch
```

Comment:
should be arranged and named according to how modules are arranged otherwise this can get messy as it increases